adapters/ix/ix.go

```diff
@@ -186,7 +186,8 @@ func (a *IxAdapter) MakeBids(internalRequest *openrtb2.BidRequest, externalReque
 	var errs []error
 
 	for _, seatBid := range bidResponse.SeatBid {
-		for _, bid := range seatBid.Bid {
+		for i := 0; i < len(seatBid.Bid); i++ {
```

Comment:
Nitpick - We can use index value returned by range of slice directly. The first value returned in case of array or slice is index. This can written simply as:-
`for i, bid := range seatBid.Bid`

Reply:
Thanks updated